app/javascript/react/ansibleCatalog/helpers.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Field } from 'react-final-form';
+import { PLAYBOOK_OPTIONS } from './constants';
+
+export const buildDropDown = (data, label, value) => {
```

Comment:
```javascript
export const buildDropDown = (data = [], label, value) => data.map(item => ({ label: item[label], value: item[value] }));
```

It will also return empty array.